Outputs violating files when running `check`

Issue came out of https://github.com/HubSpot/prettier-maven-plugin/issues/48

Currently running `check` will simply fail the goal without listing any specific **non-java** files. Would be nice to see them. 

```bash
admin@admins project % mvn prettier:check
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building project 0.1.1
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- prettier-maven-plugin:0.14:check (default-cli) @ project ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 42.415 s
[INFO] Finished at: 2021-07-14T11:50:03-04:00
[INFO] Final Memory: 14M/50M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.hubspot.maven.plugins:prettier-maven-plugin:0.14:check (default-cli) on project bridge: Error trying to run prettier-java: 1 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/src/main/java/com/hubspot/maven/plugins/prettier/CheckMojo.java
+++ b/src/main/java/com/hubspot/maven/plugins/prettier/CheckMojo.java
@@ -2,6 +2,7 @@ package com.hubspot.maven.plugins.prettier;
 
 import com.hubspot.maven.plugins.prettier.diff.DiffGenerator;
 import com.hubspot.maven.plugins.prettier.diff.GenerateDiffArgs;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.List;
@@ -37,15 +38,11 @@ public class CheckMojo extends AbstractPrettierMojo {
 
   @Override
   protected void handlePrettierLogLine(String line) {
-    if (line.endsWith(".java")) {
-      line = trimLogLevel(line);
+    line = trimLogLevel(line);
 
-      Path baseDir = project
-          .getBasedir()
-          .toPath()
-          .toAbsolutePath();
-
-      incorrectlyFormattedFiles.add(baseDir.resolve(line));
+    Path maybeFile = resolveFile(line);
+    if (Files.isRegularFile(maybeFile)) {
+      incorrectlyFormattedFiles.add(maybeFile);
       String message = "Incorrectly formatted file: " + line;
       if (fail) {
         getLog().error(message);
@@ -98,12 +95,27 @@ public class CheckMojo extends AbstractPrettierMojo {
     }
   }
 
+  private Path resolveFile(String relativePath) {
+    Path baseDir = project
+        .getBasedir()
+        .toPath()
+        .toAbsolutePath();
+
+    return baseDir.resolve(relativePath);
+  }
+
   private static String trimLogLevel(String line) {
-    if (line.contains("]")) {
-      // converts something like '[warn] src/main/java/Test.java' -> 'src/main/java/Test.java'
-      return line.substring(line.indexOf(']') + 2);
-    } else {
+    int closeBracketIndex = line.indexOf(']');
+    if (closeBracketIndex < 0) {
       return line;
     }
+
+    int startFileIndex = closeBracketIndex + "] ".length();
+    if (startFileIndex >= line.length()) {
+      return line;
+    }
+
+    // converts something like '[warn] src/main/java/Test.java' -> 'src/main/java/Test.java'
+    return line.substring(startFileIndex);
   }
 }

